src/main/flight/pid.c

```diff
@@ -349,6 +352,30 @@ bool pidInitFilters(void)
         }
     }
 
+#ifdef USE_SMITH_PREDICTOR
+    smithPredictorInit(
```

Comment:
any reason you don't use a for loop here to get all three axis?

Reply:
At the end it's almost the same. Loop or 3 calls.

Reply:
do 3 calls use more memory than a loop? Thats what i've always thought and for init code we don't need to worry about performance. Maybe i'm wrong and the compiler will optimize for that anyway.